_i18n/es/resources/user-guides/ledger-wallet-cli.md

```diff
@@ -1,165 +1,165 @@
+1. Recomendamos fuertemente probar el proceso completo primero. Esto es, enviar una pequeña cantidad al monedero y después restaurarlo (utilizando la guía ya mencionada) para verificar que puedes recuperar el monedero. Ten en cuenta que, al recrear / restaurar el monedero, deberás adjuntar la bandera `--restore-height` (con una altura de bloque anterior a la altura de tu primera transacción al monedero) al comando en el paso 3 (Windows), paso 5 (Mac OS X), o paso 3 (Linux). Más información acerca de la altura de restauración y cómo aproximarla puede encontrarse [aquí](https://monero.stackexchange.com/questions/7581/what-is-the-relevance-of-the-restore-height).
 
-2. If you use a remote node, append the `--daemon-address host:port` flag to the command in step 3 (Windows), step 5 (Mac OS X), or step 3 (Linux).
+2. Si utilizar un nodo remoto, adjunta la bandera `--daemon-address host:port` al comando en el paso 3 (Windows), paso 5 (Mac OS X), o paso 3 (Linux).
 
-3. If desired, you can manually tweak the `--subaddress-lookahead` value. The first value is the number of accounts and the second value is the number of subaddresses per account. Thus, if you, for instance, want to pregenerate 5 accounts with 100 subaddresses each, use `--subaddress-lookahead 5:100`. Bear in mind that, the more subaddresses you pregenerate, the longer it takes for the Ledger to create your wallet.
+3. Si lo deseas, puedes ajustar el valor de `--subaddress-lookahead`. El primer valor es el número de cuentas y el segundo valor es el número de subdirecciones por cuenta. Así, si por ejemplo deseas pre-generar 5 cuentas con 100 subdirecciones cada una, utiliza `--subaddress-lookahead 5:100`. Ten en mente que, entre más subdirecciones pre-generes, más tiempo le tomará a la Ledger el crear tu monedero.
 
-4. You only have to use the `--generate-from-device` flag once (i.e. upon wallet creation). Thereafter, you'd basically use it similar to how you normally use the CLI. That is:
-   1. Make sure your Ledger is plugged in and the Monero app is running.
-   2. Open `monero-wallet-cli`.
-   3. Enter the wallet name of your Ledger Monero wallet.
-   4. Enter the password to open the wallet.
+4. Sólo tienes que utilizar la bandera `--generate-from-device` una vez (e.g. sobre creación de monedero). Después de eso, básicamente lo utilizarás de manera similar a como normalmente utilizas la consola de comandos. Esto es:
+   1. Asegúrate de que tu Ledger está conectada y la aplicación Monero está corriendo.
```

Comment:
Well, it's a bit common to say it in that way here, but yes, it's more professional to use _ejecutar_. Said that, it can be changed to:

_1. Asegúrate de que tu Ledger está conectada y la aplicación Monero está en ejecución._